tamr_unify_client/_custom_encoder.py

```diff
@@ -0,0 +1,457 @@
+# flake8: noqa: C901
+"""Implementation of JSONEncoder
+"""
```

Comment:
Could you replace "Implementation of JSONEncoder" with a module-level docstrings that links to where this is adapted from? Ideally, you could also call out in the module docstring which parts of this are modified from the source.

Reply:
How's it look now?